src/Package/Impl/Repl/VsRInteractiveWorkflowProvider.cs

```diff
@@ -74,10 +79,20 @@ internal class VsRInteractiveWorkflowProvider : IRInteractiveWorkflowProvider, I
                                                     _plotsProvider, _activeTextViewTracker, _debuggerModeTracker, 
                                                     _shell, settings, _wss, () => DisposeInstance(sessionProvider));
             _disposableBag.Add(workflow);
+
+            sessionProvider.BrokerChanging += OnBrokerChanging;
             return workflow;
         }
 
+        private void OnBrokerChanging(object sender, EventArgs e) {
```

Comment:
The problem here is that workflow.ActiveWindow is null and we have to the create it, effectively duplicating the code in the show REPL command. We can alternatively add a GetOrCreateWindow to workflow but that seems to be a bit too much...